### PR TITLE
chore: update status of snackbar in README and add link to docs

### PR DESCRIPTION
**Pull request by fxck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ High level items planned for October 2016:
 | tooltip          | Initial version, needs enhancements | [README][18] |              - |
 | ripples          |  Available, but needs to be applied | [README][19] |   [#108][0108] |
 | dialog           |  Started, not yet ready for release |           -  |   [#114][0114] |
-| snackbar / toast |                    Proof-of-concept |           -  |   [#115][0115] |
+| snackbar / toast | Initial version, needs enhancements	| [README][19] |   [#115][0115] |
```
Comment (joejordanbrown) [inline README.md]: [README][19] instead of [README][21]

Comment (fxck) [inline README.md]: right, fixed, thanks

File: README.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ High level items planned for October 2016:
 | tooltip          | Initial version, needs enhancements | [README][18] |              - |
 | ripples          |  Available, but needs to be applied | [README][19] |   [#108][0108] |
 | dialog           |  Started, not yet ready for release |           -  |   [#114][0114] |
-| snackbar / toast |                    Proof-of-concept |           -  |   [#115][0115] |
+| snackbar / toast | Initial version, needs enhancements	| [README][21] |   [#115][0115] |
```
Comment (jelbourn) [inline README.md]: Can you align the column markers?

Comment (fxck) [inline README.md]: It's appears aligned in the github editor, I'm not sure what's up with the diff.. let me see what I can do about it

image

Comment (fxck) [inline README.md]: Done.

**Pull request merged**